src/Views/help.html

```diff
@@ -18,67 +18,55 @@ <h4 class="modal-title">Help</h4>
     <div style="padding-top: 10px;">
         <div id="about" ng-if="vm.selectedHelpTabName==='help'">
             <div id="helpContent">
-                <div ng-hide="vm.helpArticles"><i class="fa fa-spinner fa-3x fa-spin loadingSpinner"></i></div>
-                <accordion id="helpArticles">
-                    <accordion-group ng-repeat="article in vm.helpArticles" is-open="article.open">
-                        <accordion-heading><i ng-class="vm.helpArticles[$index].open ? 'fa fa-chevron-down': 'fa fa-chevron-right'"></i> {{article.title}}</accordion-heading>
-                        <div class="row">
-                            <div class="col-sm-12" ng-bind-html="vm.convertUnsafe(article.description)"></div>
-                        </div>
-                    </accordion-group>
-                </accordion>
+                <p>
+                    Our <a href="https://www.usgs.gov/streamstats/how-guides">how-to-guides</a> offer step-by-step instructions to guide users through the StreamStats workflow.

```

Comment:
Remove extra space & copy terminology from the Drupl page

```suggestion
                    Our<a href="https://www.usgs.gov/streamstats/how-guides">How-To Guides</a>offer step-by-step instructions to guide users through the StreamStats workflow.

```